core/src/main/java/com/datastrato/gravitino/catalog/CapabilityHelpers.java

```diff
@@ -89,6 +240,20 @@ private static TableChange applyCapabilities(
         updateColumnDefaultValue.getNewDefaultValue());
   }
 
+  private static TableChange applyCapabilities(
+      TableChange.RenameColumn renameColumn, Capability capabilities) {
+    String[] fieldName = applyCaseSensitiveOnColumnName(renameColumn.fieldName(), capabilities);
+    String newName = renameColumn.getNewName();
+    if (fieldName.length == 1) {
```

Comment:
What if length >=1?

Reply:
length >=1 means renaming nested field, not column, so no need to handle